src/fides/api/models/privacy_request.py

```diff
@@ -1737,7 +1761,11 @@ class RequestTask(Base):
     action_type = Column(EnumColumn(ActionType), nullable=False, index=True)
 
     status = Column(
-        EnumColumn(ExecutionLogStatus),  # character varying in database
+        EnumColumn(
+            ExecutionLogStatus,
+            native_enum=False,
+            values_callable=lambda x: [i.value for i in x],
+        ),  # character varying in database
```

Comment:
Same thing for "RequestTask.status" - as a reminder, these are the resources created by DSR 3.0 now that individual tasks are persisted to the database.  They share statuses with ExecutionLogs - these status updates are tightly tied together.  For example, when a DSR fails, it's RequestTask is updated to have an error status, and an ExecutionLog is created with an error status.  Both of these operations happen at the same time in GraphTsk.